src/assets/scss/3-components/3-product-control.scss

```diff
@@ -0,0 +1,20 @@
+#product-control {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  z-index: 10;
+}
+
+.product-control__icon {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  background: $green;
+  @include flex(row, center, center, warap);
+  cursor: pointer;
+
+  &:before {
+    content: "+";
```

Comment:
Could probably be implemented with just adding `+` in the markup. I could be lying, but this approach might be less accessible and would require adding an `aria` attribute to inform screen readers of what this button does.

Reply:
You are right, didnt knew that. Thanks!